docker-images/base/oss/debian/Dockerfile

```diff
@@ -17,12 +17,13 @@ RUN apt-get update && \
 ADD --chmod=755 ./get_jdk_url.sh /tmp/
 ARG jdk_version
```

Comment:
missing `ADD --chmod=755 ./get_arch.sh /tmp/`

Reply:
thanks! I haven't tested the debian images yet, still working on getting CentOS images to pass the tests.